bootstrap_cfn/config.py

```diff
@@ -91,13 +97,170 @@ def process(self):
         return json.dumps(
             template, sort_keys=True, indent=4, separators=(',', ': '))
 
+    def base_template(self):
+        from troposphere import Template
+
+        t = Template()
+
+        t.add_mapping("AWSRegion2AMI", {
+            "eu-west-1": {"AMI": "ami-f0b11187"},
+        })
+
+        if 'vpc' in self.data:
+            t.app_mapping("SubnetConfig", {
```

Comment:
Probably cos this was only 'tested' from the process function which wasn't tested well enough. I'll add more